StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+            Driver.Manage().Cookies.AddCookie(new Cookie("PVH_COOKIES_GDPR_SOCIALMEDIA", "Accept"));
+            Driver.Manage().Cookies.AddCookie(new Cookie("newsletterSeen", "true"));
+            Driver.Navigate().GoToUrl(baseURLCK);
+        }
+
+        [Given(@"I have one item in my shopping basket")]
+        public void GivenIHaveOneItemInMyShoppingBasket()
+        {
+            string ShoppingBasket1 = baseURLCK + "BasketRePopulate?ean=8719113837794&qty=1";
+            Driver.Navigate().GoToUrl(ShoppingBasket1);
+        }
+
+        [Given(@"I proceed to the checkout page")]
+        public void GivenIProceedToTheCheckoutPage()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("stickyPanel")).Displayed);
+            IWebElement CheckOutButton = Driver.FindElement(By.ClassName("stickyPanel"));
+            CheckOutButton.FindElement(By.ClassName("proceedToCheckout")).Click();
+        }
+
+        [Given(@"I continue as a CK guest")]
+        public void GivenIContinueAsACKGuest()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("guestLogin")).Displayed);
+            IWebElement GuestLogin = Driver.FindElement(By.ClassName("guestLogin"));
+            GuestLogin.FindElement(By.Id("checkoutContinueGuestWebsphere")).Click();
+        }
+
+        [When(@"Nearest UPS point is selected")]
+        public void WhenNearestUPSPointIsSelected()
+        {
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("checkoutColumn")).Displayed);
+            IWebElement RadioButton1 = Driver.FindElement(By.ClassName("deliveryAddress"));
+            var UPSRadioButton = RadioButton1.FindElements(By.ClassName("methodName"));
+            UPSRadioButton.ElementAt(0).Click();
+            Wait.Until(Driver => Driver.FindElement(By.Id("pupContainer")).Displayed);
+            Driver.SwitchTo().Frame("pupContainer");
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("iframeDialog")).Displayed);
+            IWebElement bewerken = Driver.FindElement(By.Id("editLink"));
+            bewerken.FindElement(By.Name("EditLink")).Click();
+            IWebElement InvulVelden = Driver.FindElement(By.ClassName("iframeDialog"));
+            InvulVelden.FindElement(By.Name("AddressLine1")).SendKeys("Danzigerkade 165");
+            InvulVelden.FindElement(By.Name("City")).SendKeys("Amsterdam");
+            InvulVelden.FindElement(By.Name("PostalCode")).SendKeys("1013 AP");
+            InvulVelden.FindElement(By.Name("FindButton")).Click();
+            Wait.Until(Driver => Driver.FindElement(By.ClassName("subcol5")).Displayed);
+            IWebElement LocationSelector = Driver.FindElements(By.ClassName("subcol5")).ElementAt(0);
+            LocationSelector.FindElement(By.Name("SelectLocation")).Click();
+            Driver.SwitchTo().DefaultContent();
+            System.Threading.Thread.Sleep(1000);
```

Comment:
NO.... we should avoid all `Thread.Sleep`, you could wait something happen,  or if it is really so complicated,  you should make a comment and add `TODO`, we could do it in the future, we cannot leave something like this